pkg/profiler/cpu/cpu.go

```diff
@@ -886,10 +924,14 @@ func (p *CPU) obtainProfiles(ctx context.Context) ([]*profiler.Profile, error) {
 						}
 					}
 
+					normalizedAddress, err := p.normalizer.Normalize(int(key.PID), m, addr)
+					if err != nil {
+						level.Debug(p.logger).Log("msg", "failed to normalise address", "pid", id.PID, "address", addr, "err", err)
+						continue
```

Comment:
Just realized that this is wrong, we have an inner loop but we want to skip the whole stack, so we need to continue the outer loop. 😓 

cc/ @kakkoyun

Reply:
We can always patch 🤗 Sorry, that I have missed it during reviews.